rail/topo/.htaccess

```
@@ -11,37 +11,43 @@ AddType application/ld+json .jsonld
 # Rewrite engine setup
 RewriteEngine On
 
+# Forward to most recent version
+
+RewriteRule ^$ /v1.0 [L]
```

That would try to redirect to `https://w3id.org/v1.0`.  I think you can just remove the `/` for it to rerun rules here :
```suggestion
RewriteRule ^$ v1.0 [L]
```

---

Good catch, thanks!
worked this way when I tested your suggestion locally.